layouts/partials/fragments/nav.html

```diff
@@ -111,7 +111,7 @@
   </div>
   {{- with .Params.repo_button }}
     <div class="d-none d-lg-block pl-auto">
-      <a class="btn btn-light ignore-color-change" href="{{ .url }}" role="button">
+      <a class="btn btn-light ignore-color-change" href="{{ .url }}" role="button" title="{{- i18n "navbar.github" | default "Github" -}}">
```

Comment:
Not sure, if the default "Github" makes sense.
It's mostly a Button as default or?

Reply:
I don't understand. Would you please elaborate?

Reply:
Nothing was thinking in the wrong direction. Sorry. All good.